homeassistant/components/google_air_quality/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+  "domain": "google_air_quality",
+  "name": "Google Air Quality",
+  "codeowners": ["@Thomas55555"],
+  "config_flow": true,
+  "dependencies": ["application_credentials"],
```

Comment:
I just had a look at the Google Weather integration PR. There is just the API-Key used. This would also work. So we can avoid the complicated OAuth2 workflow. Is this wanted?